### PR TITLE
[Python] Eliminate ZCLReadAttribute/ZCLSend

### PR DESCRIPTION
**Pull request by agners**
This eliminates use of ZCLReadAttribute and ZCLSend and improves some tests using asyncio.

### CODE REVIEW

**Comment by agners** [comment]
Afaict, the Darwin Tests failure are unrelated to this change (also happen on master from time to time).

File: src/controller/python/test/test_scripts/base.py [hunk 1]
```diff
@@ -429,35 +407,36 @@ def TestFailsafe(self, nodeid: int):
             pass
```
Comment (tehampson) [inline src/controller/python/test/test_scripts/base.py:407]: nit, again, I know that this is outside your PR but can you update this catch of all exceptions to catching the excepted exception (in code above)?

If that is too much a TODO with a link to github issue number is good enough for me.

File: src/controller/python/test/test_scripts/base.py [hunk 2]
```diff
@@ -387,12 +365,12 @@ def TestFailsafe(self, nodeid: int):
...
+            await self.devCtrl.SendCommand(
                 nodeid,
                 0,
                 Clusters.AdministratorCommissioning.Commands.OpenBasicCommissioningWindow(180),
                 timedRequestTimeoutMs=10000
-            ))
+            )
             # we actually want the exception here because we want to see a failure, so return False here
             self.logger.error(
                 'Incorrectly succeeded in opening basic commissioning window')
```
Comment (tehampson) [inline src/controller/python/test/test_scripts/base.py:376]: nit, I know that this is outside your PR but can you update this catch of all exceptions to catching the excepted exception (in code below)?

If that is too much maybe add a TODO with a link to github issue number

**Review by tehampson — APPROVED** [review]

File: src/controller/python/test/test_scripts/base.py (hunk 1 above)
Comment (agners) [reply src/controller/python/test/test_scripts/base.py:407]: Yeah looked at that too and thought about it :smile: 

Did just that now.

File: src/controller/python/test/test_scripts/base.py (hunk 2 above)
Comment (agners) [reply src/controller/python/test/test_scripts/base.py:376]: And here too!

**Review by cecille — COMMENTED** [review]
Is the plan to remove these from ChipDeviceCtrl as well?

**Review by cecille — APPROVED** [review]

**Comment by agners** [comment]
> Is the plan to remove these from ChipDeviceCtrl as well?

Yes, along with `chip-device-ctrl` (which currently still uses these APIs).

**Review by fessehaeve — APPROVED** [review]

**Pull request merged**